io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/XLSUtil.java

```diff
@@ -30,15 +37,23 @@
  */
 public class XLSUtil {
 
+	/**
+	 * Default Time Formatter
+	 */
+	public static final String PARAMETER_DATE_FORMAT = "yyyy-MM-dd";
```

Comment:
Maybe rename to `PARAMETER_DATE_FORMAT_DEFAULT` or similar to avoid using the same name than for the constant with the actual parameter name?

Reply:
On a second look - is this actually used anymore?

Reply:
Not used anymore. I have remove it